cli.go

```diff
@@ -1170,6 +1449,38 @@ func GithubListReposByLanguage(owner string, lang string) ([]*github.Repository,
 
 	return repos, nil
 }
+func ListAllReposByLanguage(lang string) ([]*github.Repository, error) {
+	lang = strings.TrimSpace(lang)
+
+	opts := &ghc.ListAllReposByLanguageOpts{
+		Language:     lang,
+		ExcludeForks: true,
+		// Limit:        limit, // Maybe add a `--limit=n` flag?
+	}
+	repos, err := ghClient.ListAllReposByLanguage(opts)
+	if err != nil {
+		return nil, err
+	}
+
+	return repos, nil
+}
+func GithubListReposByMetaSearch(query string) ([]*github.Repository, error) {
+	return ghClient.SearchRepos(query)
+}
+func GithubListReposByCodeSearch(query string) ([]*github.Repository, error) {
+	var repos []*github.Repository
+
+	codeResults, err := ghClient.SearchCode(query)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
To use `DeduplicateSlice2`, you'll need to update the `github.com/gagliardetto/utilz` module version used in `lgtm-cli`.

```suggestion

		// Deduplicate results (for any given repo, there might be more than one code results).
		DeduplicateSlice2(&codeResults, func(i int) string {
			return codeResults[i].Repository.GetHTMLURL()
		})

```